parsl/executors/workqueue/executor.py

```diff
@@ -406,13 +408,17 @@ class WorkQueueExecutor(NoStatusHandlingExecutor):
 
         project_name: str
             If given, Work Queue master process name. Default is None.
-
-        project_password: str
-            Optional password for the Work Queue project. Default is None.
+            Overrides hostname.
 
         project_password_file: str
             Optional password file for the work queue project. Default is None.
 
+        hostname: str
```

Comment:
Minor. Might be useful to have this match keywords used in other executors, in this case `address`. Probably best to push to next round of dev.

Reply:
Sound good, it is an easy change.  I'll have a pr ready for when this one is merged.